Update Flizzer tracker app

LTVA1 opened the pull request:
# Application Submission

- Fix build for new firmware version


# Extra Requirements 

- [ Describe if your application requires any extra hardware or software ]


# Author Checklist (Fill this out)

- [x] I've read the [contribution guidelines](../blob/HEAD/documentation/Contributing.md) and my PR follows them
- [x] I own the code I'm submitting or have code owner's permission to submit it
- [x] I [have validated](../blob/HEAD/documentation/Contributing.md#validating-manifest) the manifest file(s) with `python3 tools/bundle.py --nolint applications/CATEGORY/APPID/manifest.yml bundle.zip`


# Reviewer Checklist (Don't fill this out)

- [x] Bundle is valid
- [x] There are no obvious issues with the source code
- [x] I've ran this application and verified its functionality

LTVA1 commented:
Builds on dev branch. I made a PR because Furippa told me it doesn't build for rc branch. Try for the branch where CLI refactor is present

doomwastaken commented:
Yeah, no worries, I will merge once those changes are merged

doomwastaken commented:
Just checked and in both cases cli.h is not included into SDK, so you would need to include it yourself

LTVA1 commented:
What do you mean? It built on dev branch two days ago... Do I also need to add `include <cli/cli.h>`? Did you return it into SDK?

LTVA1 commented:
@doomwastaken

doomwastaken commented:
As far as I understood and looked through files, you want to include cli.h, which in order includes cli_registry. And old headers are no longer are included into the SDK (the ones that we got errors in the action) 

@hedger Could you clarify this please

doomwastaken commented:
https://github.com/flipperdevices/flipperzero-firmware/blob/dev/applications/services/cli/cli.h

LTVA1 commented:
> As far as I understood and looked through files, you want to include cli.h, which in order includes cli_registry. And old headers are no longer are included into the SDK (the ones that we got errors in the action)

I removed `<cli/cli.h>` include because that was the error I got from furippa. It said that in `rc` branch the file doesn't exist. When I compiled on dev branch it said the same thing. Thus I replaced the include with `<cli/cli_main_shell.h>` which did successfully compile for dev branch and did successfully work on dev branch firmware which I flashed for testing. The same thing for my other PR with video player app. Thus I don't understand why it doesn't build now because you said you merged the CLI refactor into the whatever branch the bot compiles app for. @doomwastaken

LTVA1 commented:
Here in compilation errors it says that `<cli/cli_main_shell.h>` does not exist. Why? Maybe you set it up to compile for different branch than the one the CLI refactor is present in?

portasynthinca3 commented:
@LTVA1 Please try running `ufbt update -c rc` and update us.

LTVA1 commented:
I use fbt

LTVA1 commented:
I will still try though

portasynthinca3 commented:
Unfortunately, `cli_main_shell.h` is not part of the SDK. It's not supposed to be, it doesn't provide any functionality that would be useful to external apps. I searched your repo for any uses of `cli_` and couldn't find any. Can you please tell us what you're trying to accomplish with this include? If it _is_ unused, it's best to just remove it.

LTVA1 commented:
I just kept it all the way through the development of both apps since these were my first and only two apps for Flipper. I will try to remove these includes, most probably nothing would break indeed.

LTVA1 commented:
@doomwastaken CLI include removed there and in video player app. Both apps were reocmpiled and confirmed to work on rc branch. Although the build may still fail since I use fbt (bc I don't like cut-down versions of tools) and your CI/CD pipeline seems to use the ufbt
